examples/js/modifiers/BendModifier.js

```diff
@@ -0,0 +1,149 @@
+ /** The Bend modifier lets you bend the current selection up to 90 degrees about a single axis,
+ * producing a uniform bend in an object's geometry.
+ * You can control the angle and direction of the bend on any of three axes.
+ * The geometry has to have rather large number of polygons!
+ * options:
+ * 	 direction - deformation direction (in local coordinates!). 
+ * 	 axis - deformation axis (in local coordinates!). Vector of direction and axis are perpendicular.
+ * 	 angle - deformation angle.
+ * @author Vildanov Almaz / alvild@gmail.com
+ * The algorithm of a bend is based on the chain line cosh: y = 1/b * cosh(b*x) - 1/b. It can be used only in three.js.
+ */
+
+THREE.BendModifier = function () {
+
+};
+
+THREE.BendModifier.prototype = {
+
+    constructor: THREE.BendModifier,
+
+    set: function ( direction, axis, angle ) {
+        this.direction = new THREE.Vector3(); this.direction.copy( direction );
+		this.axis = new THREE.Vector3(); this.axis.copy( axis );
+        this.angle = angle;
+        return this
+    },
+
+	_sign: function (a) {
+        return 0 > a ? -1 : 0 < a ? 1 : 0
+    },
+
+	_cosh: function( x )  {
+		return ( Math.exp( x ) + Math.exp( -x ) ) / 2;
+	},
+
+	_sinhInverse: function( x )  {
+			return  Math.log( Math.abs( x ) + Math.sqrt( x * x + 1 ) );
+	},
+
+    modify: function ( geometry ) {
+
+		var thirdAxis = new THREE.Vector3();  thirdAxis.crossVectors( this.direction, this.axis );
+
+		// P - matrices of the change-of-coordinates
+		var P = new THREE.Matrix4();
+		P.set ( thirdAxis.x, thirdAxis.y, thirdAxis.z, 0, 
+			this.direction.x, this.direction.y, this.direction.z, 0, 
+			this.axis.x, this.axis.y, this.axis.z, 0, 
+			0, 0, 0, 1 ).transpose();
+
+		var InverseP =  new THREE.Matrix3().getInverse( P );
+		var newVertices = []; var oldVertices = []; var anglesBetweenOldandNewVertices = [];
+
+		var meshGeometryBoundingBoxMaxx = 0; var meshGeometryBoundingBoxMinx = 0;
```

Comment:
I think you should use [Box2](https://github.com/mrdoob/three.js/blob/master/src/math/Box2.js) here.